antismash/outputs/html/__init__.py

```diff
@@ -60,19 +62,36 @@ def write(records: List[Record], results: List[Dict[str, ModuleResults]],
     """
     output_dir = options.output_dir
 
-    copy_template_dir('css', output_dir)
+    copy_template_dir('css', output_dir, pattern="%s.css" % options.taxon)
     copy_template_dir('js', output_dir)
     copy_template_dir('images', output_dir)
 
     # Generate structure images for records obtained from BioSQL
     generate_webpage(records, results, options)
 
 
-def copy_template_dir(template: str, output_dir: str) -> None:
+def copy_template_dir(template: str, output_dir: str, pattern: Optional[str] = None) -> None:
     """ Copy files from a template directory to the output directory, removes
-        any existing directory first
+        any existing directory first. If pattern is supplied, only files within
+        the template directory that match the template will be copied.
+
+        Arguments:
+            template: the source directory
+            output_dir: the target directory
+            only_matching: a pattern to restrict to, if given
```

Comment:
Shouldn't this be `pattern`?

Reply:
It should be, yes. I just fixed that now.